service/routes.py

```diff
@@ -55,6 +60,7 @@ def get_inventory_records(product_id):
     app.logger.info("Returning product: %s", product.name)
     return jsonify(product.serialize()),status.HTTP_200_OK
 
+@app.route("/inventory-records/", methods=["POST"])
```

Comment:
```
app = Flask('my_app')
app.url_map.strict_slashes = False```
Instead, can we do this? https://stackoverflow.com/questions/40365390/trailing-slash-in-flask-route

Reply:
Good suggestion. I will follow this